src/Spice86.Shared/Utils/ConvertUtils.cs

```diff
@@ -362,7 +362,7 @@ public static class ConvertUtils {
     /// <param name="path">The path string to modify.</param>
     /// <returns>A new string with all backslashes replaced with forward slashes.</returns>
     public static string ToSlashPath(string path) {
-        return path.Replace('\\', '/').Replace("//", "/");
+        return path.Replace('\\', Path.DirectorySeparatorChar).Replace("//", Path.DirectorySeparatorChar.ToString());
```

Comment:
Then maybe function name needs to change because on windows it will be \
Or maybe it is better to keep / since it would make emulator behavior the same for any host OS?
Not sure what is best.
Why you needed to change this by the way.

Reply:
I reverted this cosmetic change which had no real value.